### ISSUE TITLE
Date range graph issues

### ISSUE THREAD

**Issue by ODOT-RPTD-mb** [ODOT-PTS/TNAST_MAVEN#94]
"gtfs" feed shows up in graph - no text to tie to any agency?
cccexpress and eugenetobend feeds end prematurely?
@PPaulsonOregonDOT 

![image](https://user-images.githubusercontent.com/8129162/36985413-0dd09f4a-204c-11e8-8adb-ba7a70c82199.png)

**Comment by irees** [comment]
@ODOT-RPTD-mb the "gtfs" feed is TriMet; it was based on the url in the spreadsheet. I'll make sure it is renamed to something more appropriate in the next build.

**Comment by irees** [comment]
@KalonThomas Can you check the end dates for `cccexpress` and `eugenetobend`?

**Comment by KalonThomas** [comment]
@irees 

`cccxpress` The feed end date is correct in this case (20180321). I'm making an inquiry with the agency for next semesters schedule info and can let you know when updates to the GTFS have been made extending the feed.

`eugenetobend` it looks like data was being pulled from an old url prior to a few agencies merged into Pacific Crest Lines. The data url has been updated to in the [ODOT GTFS feeds](https://docs.google.com/spreadsheets/d/1Pq2vGSW3ba_Y-xvf80aIcB8dim6c6wr_-JnawyJGm7k/edit#gid=0) spreadsheet
to: http://oregon-gtfs.com/gtfs_data/pacificcrest-or-us/pacificcrest-or-us.zip.

**Comment by irees** [comment]
I have updated with the most recent set of feeds in Kalon's spreadsheet above. A few urls had the same or non-descriptive filenames, so I assigned them a filename based on the "agency name" field in the spreadsheet.

http://tna.trilliumtransit.com:8090/TNAtoolAPI-Webapp/Daterange.html?dbindex=11

Note: a few smaller feeds failed to import due to some database uniqueness constraint being violated - looking into it.
